trino-lb/src/http_server/mod.rs

```diff
@@ -121,6 +122,15 @@ pub async fn start_http_server(
             "/v1/statement/executing/{query_id}/{slug}/{token}",
             delete(v1::statement::delete_trino_executing_statement),
         )
+        .route(
+            "/admin/activate-cluster/{cluster_name}",
+            post(admin::post_activate_cluster),
+        )
+        .route(
+            "/admin/deactivate-cluster/{cluster_name}",
+            post(admin::post_deactivate_cluster),
+        )
+        .route("/admin/cluster-status", get(admin::get_cluster_status))
```

Comment:
note: I think it makes sense to not even add these routes if no admin credentials are configured. This would also remove the need for the `NoAdminAuthenticationMethodDefined` error variant above.